modules/common/src/main/scala/surge/internal/utils/DiagnosticContextFuturePropagation.scala

```diff
@@ -14,15 +15,20 @@ class DiagnosticContextFuturePropagation(executionContext: ExecutionContext) ext
   override def execute(runnable: Runnable): Unit = {
     val callerMdc = Option(MDC.getCopyOfContextMap) // get a copy of the MDC context data
     val callerSpan = Span.current()
+    println(s"JEFF --- caller MDC = $callerMdc")
+    println(s"JEFF --- caller span = $callerSpan")
```

Comment:
A bunch of these printlns will need to be cleaned up. I was using for debug but they ultimately don't belong in the code